src/org/thoughtcrime/securesms/loki/redesign/activities/HomeActivity.kt

```diff
@@ -230,8 +233,8 @@ class HomeActivity : PassphraseRequiredActionBarActivity, ConversationClickListe
         @SuppressLint("StaticFieldLeak")
         override fun onSwiped(viewHolder: RecyclerView.ViewHolder, direction: Int) {
             val threadID = (viewHolder as HomeAdapter.ViewHolder).view.thread!!.threadId
+            val recipient = (viewHolder as HomeAdapter.ViewHolder).view.thread!!.recipient
```

Comment:
Casting to `HomeAdapter.ViewHolder` is redundant here as the line above already did it